almost-jira/src/main/java/pl/edu/pw/elka/pap/z16/almostjira/controllers/tasks/TasksController.java

```diff
@@ -29,26 +27,26 @@ public TasksController(ProjectService projectService) {
     public ResponseEntity<Object> addTaskToProject(@PathVariable("id") String projectId, String newTask){
         try {
             var modifiedList = projectService.getTasks(projectId);
-            if (modifiedList == null)
-                modifiedList = new ArrayList<>();
-            if (newTask == null || newTask.equals(""))
-                return ResponseHandler.generateResponse("empty task", HttpStatus.OK, projectService.updateProjectUpdateTasks(modifiedList, projectId));
+
+            // FIXME: Adding empty task results in OK response but we won't add this task to the project
```

Comment:
this should result in different status code and requires more work before merging